README.md

```diff
@@ -13,15 +13,42 @@ Usage of this gem is pretty straight-forward. Add it to your bundle like this:
 gem 'jekyll-github-metadata'
 ```
 
-Then go ahead and run `bundle install`. Once you've done that, add your repo & the gem to your `_config.yml`:
+Now add it to your `_config.yml`:
 
 ```yaml
-# or use PAGES_REPO_NWO in the env
-repository: me/super-cool-project
 gems: ['jekyll-github-metadata']
 ```
 
-Then run `jekyll` like you normally would and your `site.github.*` fields should fill in like normal.
+Then go ahead and run `bundle install`. Once you've done that jekyll-github-metadata will run when you run Jekyll.
+
+In order for jekyll-github-metadata to know what metadata to fetch it must
+be able to determine the repository to ask GitHub about.
+
+The easiest way to accomplish this is by setting an "origin" remote with a
```

Comment:
I'd note that if you've run `git clone` from GitHub, this is almost 100% set, no further action needed.